Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -394,7 +405,8 @@ extension PostgresConnection {
             let configuration = PostgresConnection.InternalConfiguration(
                 connection: .resolved(address: socketAddress, serverName: serverHostname),
                 authentication: nil,
-                tls: tls
+                tls: tls,
+                requireBackendKeyData: requireBackendKeyData
```

Comment:
Just set to true. See above comment.
```suggestion
                requireBackendKeyData: true
```